src/models/pod.js

```diff
@@ -1,47 +1,44 @@
-let users = {
-  1: {
-    id: "1",
-    username: "Robin X",
-  },
-  2: {
-    id: "2",
-    username: "Carol S",
-  },
-};
 
-let messages = {
-  1: {
-    id: "1",
-    text: "Good Day",
-    userId: "1",
-  },
-  2: {
-    id: "2",
-    text: "By World",
-    userId: "2",
-  },
-};
+const { Sequelize, Model, DataTypes } = require('sequelize');
+const sequelize = new Sequelize('sqlite::memory:');
 
-// const mongoose = require("mongoose");
-// const Schema = mongoose.Schema;
+class Pod extends Model {}
+Pod.init({
+  id: DataType.STRING,
+  podname: DataType.STRING
+  rating: DataType.STRING,
```

Comment:
This only provides a single rating for the picture, but we need to store a rating per user. The rating should be a many-to-many object.